src/cascadia/TerminalSettingsModel/EnumMappings.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.

```

Comment:
nit: I would add an abstract here about what these maps are for and how they're used.